kupfer/plugin/id2.py

```diff
@@ -0,0 +1,126 @@
+""" ID2 – Guess Identifier (Kupfer Plugin). See https://id2.dev
+"""
+__kupfer_name__    = ("Identifier Resolver")
+__kupfer_sources__ = ()
+__kupfer_actions__ = ( "LookupID", )
+__description__    = ("""Detect and open DOIs, ISBNs, OrcIDs, crypto-addresses, geo-coordinates, and more in your web-browser. Examples:
+
+- Unicode:  \t"U+1F194"
+- UnixTime: \t"1606324253"
+- Location: \t"8FWH4HX8+QR"
+- DOI:    \t\t"10.1109/FDL.2018.8524068"
+- OrcID:    \t"0000-0002-0006-7761"
+- BitCoin:  \t"3FZbgi29cpjq2GjdwV8eyHuJJnkLtktZc5"
+
+For a full list, see https://id2.dev
+
+To disable a certain ID, put "x" into URL field below.
```

Comment:
Yes, should I update that and if yes, should I update the description or the behavior? 
I just thought "put x" is a simple instruction and "len < 3" is a robust implementation because no valid URL has less than 3 chars.